source/custom_templates/_tx_api_page.txt

```diff
@@ -1,8 +1,9 @@
 .. include:: _:page:.tmp
 
-.. include:: common_response_params.rst
+.. include:: _:org_type:_param_table.rst
 
-.. include:: optional_fields.txt
+The following fields are optional based on your account type:
 
+.. include:: _:org_type:_optional_param_table.rst
 
```

Comment:
No, that is not correct.

1) Call center integration params are optional (only if `cpa_enabled?`: https://github.com/Invoca/web/blob/master/app/models/reports/detail/detail_report.rb#L679-L689)

2) Signal params are optional (and only for Network and Advertiser)
https://github.com/Invoca/web/blob/master/app/models/reports/detail/detail_report.rb#L564-L567 (ends up checking a feature flag)

So what I had in my above comment should be accurate (in that these two tables would go in optional sections). It looks like neither are shown for Affiliates though, hopefully that doesn't add much work..